examples/publish/main.go

```diff
@@ -0,0 +1,64 @@
+package main
```

Comment:
since we have one example why not `examples/main.go` or add the second example for subscriber `examples/subscriber/main.go`

Reply:
the subscriber example would not be used anywhere so it is not needed and the publisher dir also serves to explain the use of the example